Assets/Scripts/BombSelector.cs

```diff
@@ -0,0 +1,50 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class BombSelector : MonoBehaviour
+{
+    public GameObject[] bombPrefabs;
+    private List<GameObject> currentBombs = new List<GameObject>();
+    private BombIndicator bombIndicator;
+    public int maxBombs;
+
+    void Start()
+    {
+        bombIndicator = GetComponentInChildren<BombIndicator>();
+        for (int index = 0; index < maxBombs; index++)
+        {
+            PopulateNextBomb(index);
+        }
+    }
+
+    public GameObject GetNextBomb()
+    {
+        GameObject nextBomb = DequeueBomb();
+        PopulateNextBomb(2);
+        return nextBomb;
+    }
+
+    private GameObject ChooseNextBomb()
+    {
+        return bombPrefabs[Random.Range(0, bombPrefabs.Length)];
+    }
+
+    private void PopulateNextBomb(int index)
```

Comment:
I dont think we use the `index` passed in

Reply:
Right, it was leftover from before.